src/storage.rs

```diff
@@ -1233,21 +1234,33 @@ fn is_known_criteria(valid_criteria: &[CriteriaName], criteria_name: &CriteriaNa
 /// which could not be interpreted, due to issues such as being created with a
 /// newer version of cargo-vet.
 pub(crate) fn foreign_audit_file_to_local(
+    name: &str,
     foreign_audit_file: ForeignAuditsFile,
 ) -> ForeignAuditFileToLocalResult {
     let mut ignored_criteria = Vec::new();
-    let mut criteria: SortedMap<CriteriaName, CriteriaEntry> = foreign_audit_file
-        .criteria
-        .into_iter()
-        .filter_map(|(criteria, value)| match parse_imported_criteria(value) {
-            Some(entry) => Some((criteria, entry)),
-            None => {
-                ignored_criteria.push(criteria);
-                None
-            }
-        })
-        .collect();
-    let valid_criteria: Vec<CriteriaName> = criteria.keys().cloned().collect();
+    let valid_criteria: Vec<CriteriaName>;
+    let mut criteria: SortedMap<CriteriaName, CriteriaEntry>;
+    if foreign_audit_file.criteria.is_empty() {
+        warn!("No criteria definitions found when importing from `{name}`");
```

Comment:
I don't think this is a situation where we want to warn. We expect most organizations using cargo-vet to not need additional custom criteria beyond `safe-to-run` and `safe-to-deploy`, so there being no criteria definitions is an expected normal state.

Reply:
Thanks for catching this.  I didn't think about the built-in `safe-to-run` and `safe-to-deploy` criteria when adding this warning.